frontend/utils/breakpoints.js

```diff
@@ -0,0 +1,3 @@
+// Same as CSS breakpoints at _variables.scss
+export const TABLET = 769
+export const DESKTOP = 1200
```

Comment:
Would `frontend/views/utils/breakpoints.js` be a better place for this file, since this is only ever used with views?

Reply:
I also thought that, but then, in order for us to do a reusable responsive resize, I believe we would need to use the store or sbp somehow... and that would affect model folder, that's why I kept it here.

Reply:
I have no idea what you mean here:

> but then, in order for us to do a reusable responsive resize, I believe we would need to use the store or sbp somehow...

Reply:
Read [this comment](https://github.com/okTurtles/group-income-simple/pull/777/files#diff-c3d9b27ac7e3a8165f584c6325634d49R115-R121). I might help you to understand what I mean

Reply:
Update: Direct quote from my changes

```js
mounted () {
    // TODO - Create a single resize listener to be reused on components
    window.addEventListener('resize', this.config.debounceResize)
  },
  beforeDestroy () {
    window.removeEventListener('resize', this.config.debounceResize)
  },
```

Reply:
I still don't see what this has to do with the store or sbp or what the model folder has to do with it?

Reply:
For now, nothing. But in the future, we'll need to implement a single resize listener that components would use and the way to do that is by using the store. But if you want I can move it to views/utils and later move it again.

Reply:
> But if you want I can move it to views/utils and later move it again.

Yeah, that would be helpful I think for now. I'll go ahead and test out this PR right now while waiting for this change -- everything else looks good to me so far!

Reply:
done